src/components/problem/edit.vue

```diff
@@ -1,7 +1,9 @@
 <template>
-  <div>
+  <div id="edit-content">
```

Comment:
If there is many edit component on page, `id` won't work, class is a better choice, isn't it?

Reply:
Oh, I forgot to delete that line, I used to want to make the height of the outbox fixed, but failed

Reply:
Fixed